package.json

```diff
@@ -19,18 +19,19 @@
         "lint": "eslint src/**/*.{ts,tsx}",
         "typeCheck": "tsc -p tsconfig.json --noEmit",
         "update-viewer": "npm i @aics/simularium-viewer@latest",
-        "test": "BABEL_ENV=commonjs TS_NODE_FILES=true TS_NODE_PROJECT=tsconfig.commonjs.json NODE_ENV=production mocha --opts mocha.opts src/**/*.test.ts"
+        "test": "jest --coverage"
     },
     "devDependencies": {
         "@babel/cli": "7.2.3",
-        "@babel/core": "7.3.3",
+        "@babel/core": "^7.8.0",
```

Comment:
This update was necessary for one of the new dependencies, `babel-jest` I believe